src/test/regress/input/multi_alter_table_statements.source

```diff
@@ -162,7 +162,54 @@ FROM
 ORDER BY attnum;
 \c - - - :master_port
 
+-- check we don't intercept DDL commands if propagation is turned off
```

Comment:
Quick format feedback.

- For comments, we start with capital letters.
- I'm not a grammar specialist but missing that :) "check that", "verify that"
- Missing newlines between blocks. Generally, there is a new line between comment blocks